packages/ds-healthcare-gov/src/components/Header/defaultMenuLinks.test.js

```diff
@@ -2,95 +2,71 @@ import defaultMenuLinks, {
   defaultMenuLinks as namedExportDefaultMenuLinks,
 } from './defaultMenuLinks';
 
-const t = (key) => key;
-
 describe('MenuList', function () {
   it('includes a named export', () => {
     // Some apps may need to import the default menu links in order to extend them
     expect(typeof namedExportDefaultMenuLinks).toBe('function');
   });
 
   it('leaves out login link if hideLoginLink true', () => {
-    expect(
-      defaultMenuLinks(t, undefined, undefined, undefined, undefined, undefined, true)
-    ).toMatchSnapshot();
+    expect(defaultMenuLinks({ hideLoginLink: true })).toMatchSnapshot();
```

Comment:
These unit tests were wrong in several places because one of the parameters in this long list would be missing. It was so hard to read!